app/assets/stylesheets/common/d-editor.scss

```
@@ -282,6 +284,15 @@
       }
     }
 
+    .delete-image-button {
+      cursor: pointer;
+      color: var(--danger);
+
+      svg {
+        padding-left: 0.5em;
+      }
```

```suggestion
      .d-icon-trash-alt {
        margin-left: 0.5em;
      }
```

The result is the same, but I think it's safer to use margins with svgs (and clearer to readers when targetting the element by class).